scripts/Packages/Package_Gullwing__QFP_SOIC_SO/size_definitions/sop.yaml

```diff
@@ -79,6 +79,51 @@ SOP-8_3.76x4.96mm_P1.27mm:
   num_pins_x: 0
   num_pins_y: 4
 
+SOP-8_3.9x4.9mm_P1.27mm:
+  size_source: 'http://www.macronix.com/Lists/Datasheet/Attachments/7534/MX25R3235F,%20Wide%20Range,%2032Mb,%20v1.6.pdf'
```

Comment:
add #page=79 to the link to point directly to the correct datasheet page.

Reply:
Done